PagingSample/app/src/main/java/paging/android/example/com/pagingsample/CheeseViewModel.kt

```diff
@@ -78,7 +78,8 @@ class CheeseViewModel(private val dao: CheeseDao) : ViewModel() {
                     } else if (before == null) {
                         // Header
                         CheeseListItem.Separator(after.name.first())
-                    } else if (before.name.first() != after.name.first()) {
+                    } else if (before.name.first().toLowerCase() !=
```

Comment:
Can we simplify this to `.equals(ignoreCase = true)`?

See: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/equals.html#equals